.github/workflows/action-test.yml

```diff
@@ -21,7 +21,7 @@ on:
   push:
     branches:
     - main
-    - unpin_iris
+    - fix-python-3.7-tests
```

Comment:
Remove this line?

Reply:
yup, good catch I just cleaned up all the GA workflow files, some had older merged+deleted branches that were supposed to be running on :+1: